framework/src/modules/chain/transaction_pool/transaction_pool.js

```diff
@@ -91,30 +91,7 @@ class TransactionPool extends EventEmitter {
 		this.processTransactions = transactions =>
 			this.blocks.processTransactions(transactions);
 
-		const poolConfig = {
-			expireTransactionsInterval: this.expireTransactionsInterval,
-			maxTransactionsPerQueue: this.maxTransactionsPerQueue,
-			receivedTransactionsLimitPerProcessing: this.bundleLimit,
-			receivedTransactionsProcessingInterval: this.bundledInterval,
-			validatedTransactionsLimitPerProcessing: this.bundleLimit,
-			validatedTransactionsProcessingInterval: this.bundledInterval,
-			verifiedTransactionsLimitPerProcessing: this.maxTransactionsPerBlock,
-			verifiedTransactionsProcessingInterval: this.bundledInterval,
-			pendingTransactionsProcessingLimit: this.maxTransactionsPerBlock,
-		};
-
-		const poolDependencies = {
-			validateTransactions: this.validateTransactions,
-			verifyTransactions: this.verifyTransactions,
-			processTransactions: this.processTransactions,
-		};
-
-		this.pool = new pool.TransactionPool({
-			...poolConfig,
-			...poolDependencies,
-		});
-
-		this.subscribeEvents();
+		this.resetPool();
```

Comment:
is `resetPool` a public interface?

Reply:
no its not public

Reply:
Could you rename it to _resetPool and the same in the below comment?